src/topology/algebra/infinite_sum.lean

```diff
@@ -73,6 +73,9 @@ by simp [has_sum, tendsto_const_nhds]
 
 lemma summable_zero : summable (λb, 0 : β → α) := has_sum_zero.summable
 
+lemma summable_empty [is_empty β] : summable f :=
+by convert summable_zero; ext s; exfalso; exact h (nonempty.intro s)
```

Comment:
I expect you'll have to adjust the proof to work with `is_empty`

Reply:
This is a nice example of why you shouldn't use `;`. The last half of your proof is nonsense as `h` doesn't even exist, but lean doesn't care because you have no goals left by the time you get there.

If you switch to commas lean will complain if you try to keep going when you're done, and then you can remove any proof that remains 
```suggestion
by { convert summable_zero, ext s, exfalso, exact h (nonempty.intro s) }
```

Reply:
I suspect all you need is
```suggestion
by convert summable_zero
```

Reply:
Thanks. That's in the latest commit.